test/int/breakOnLoad.test.ts

```diff
@@ -0,0 +1,426 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import * as path from 'path';
+const {createServer} = require('http-server');
+
+import {DebugClient} from 'vscode-debugadapter-testsupport';
+import * as ts from 'vscode-chrome-debug-core-testsupport';
+
+import * as testSetup from './testSetup';
+
+suite('BreakOnLoad', () => {
+    const DATA_ROOT = testSetup.DATA_ROOT;
+
+    suite('Regex Approach', () => {

```

Comment:
Are these tests exactly the same as the Instrument tests? Then you could parameterize them and remove the duplication. You can call a function with "regex" or "instrument" that registers the tests with the right type.

Reply:
Not exactly. Like Instrument has the suite for a 'webpack project' and regex does not. Do you still think this is a good idea?

Reply:
For the tests that are the same, I'd rather see those factored out instead of duplicated.